pkg/api/routes.go

```diff
@@ -149,5 +155,11 @@ func (h *Handler) GetRoutes() []Route {
 			"/log",
 			h.SetLogLevel,
 		},
+		{
+			"GetMetrics",
```

Comment:
See https://github.com/rook/rook/pull/472#discussion_r105769697, I believe it is idiomatic Prometheus to return all metrics from one call/route.